src/blk/pmem/PMEMDevice.cc

```diff
@@ -168,13 +243,30 @@ bool PMEMDevice::support(const std::string &path)
 {
   int is_pmem = 0;
   size_t map_len = 0;
-  void *addr = pmem_map_file(path.c_str(), 0, PMEM_FILE_EXCL, O_RDONLY, &map_len, &is_pmem);
+  void *addr;
```

Comment:
please define variable close to where it is used for the first time for better readability.

Reply:
ack